src/main/java/games/strategy/engine/lobby/server/UserManager.java

```diff
@@ -36,6 +36,9 @@ public String updateUser(final String userName, final String emailAddress, final
     }
     final HashedPassword password = new HashedPassword(hashedPassword);
     if (!password.isValidSyntax()) {
+      //TODO: decrypt hashedPassword and pass it to updateUser
+      //new DbUserController().updateUser(user, decryptedPassword);
```

Comment:
Because it introduces a minor issue and duplicates some code.
Users can already be updated using the code inside LobbyValidator, but this code uses RMI to update users which makes it almost impossible to change this code without breaking compatibility.
Also this comment should be encrypt